### PR TITLE
dealing with string arrays marginally better

### PR DESCRIPTION
**Pull request by danturn**
- fix release (#29)
- include mac
- fix release (#30)
- dont strip empty strings from end of line
- dont strip empty strings from end of line
- Performance tweaks (#31)
- sorted array content splitting
- merging with alex's optimisations

### CODE REVIEW

File: src/parsers/sanitiser.rs [hunk 1]
```diff
@@ -1,7 +1,33 @@
 pub fn trim(line: &str) -> &str {
-    line.trim()
+    line.trim_matches(|c| c == ' ' || c == '\n')
```
Comment (danturn) [inline src/parsers/sanitiser.rs:2]: previously we stripped all whitespace (including tabs)
but... empty string chars at the end of aline look like
`'asdasda'\t\n`

so the tab char got cut

**Comment by danturn** [comment]
ill switch the logging in this loop to trace level logging i think cos it was pretty useful to debug the oddities

File: src/parsers/transformer.rs [hunk 2]
```diff
@@ -134,28 +135,34 @@ fn array_string_to_vec(value: &str) -> Vec<String> {
...
+            last_char_seen,
+            i,
+            last_char_index
         );
     }
-    println!("\noutput - {:?}  ", list_output);
-    list_output
+    trace!("\noutput - {:?}", word_acc);
+    //Remove the trailing comma from line: 145!
+    word_acc[0..word_acc.len() - 1].to_string()
```
Comment (rohan-b99) [inline src/parsers/transformer.rs:165]: .pop() would be more efficient

File: src/parsers/transformer.rs [hunk 3]
```diff
@@ -134,28 +135,34 @@ fn array_string_to_vec(value: &str) -> Vec<String> {
...
-                || !word_is_quoted && c == '}')
+                || (!word_is_quoted && c == '}'))
         {
             inside_word = false;
             word_is_quoted = false;
-            list_output.push(output);
-            output = "".to_string();
-            println!("its the end of a word");
+            let transformed = transform(rng, &current_word, sub_type, transformer, table_name);
+            word_acc = format!("{}\"{}\",", word_acc, transformed);
```
Comment (rohan-b99) [inline src/parsers/transformer.rs:145]: I think .push_str would be more efficient here to append rather than creating a whole new string

**Review by rohan-b99 — APPROVED** [review]

**Pull request merged**